### ISSUE TITLE
No kfmd/openapi_client 

### ISSUE THREAD

**Issue by jinchihe** [kubeflow/metadata#91]
/kind bug

**What steps did you take and what happened:**
During I try to install and run the sdk demo cases, I found no kfmd/openapi_client, more checked, that should be generated by `make swagger-py-client` and ignored in `gitignore`.

```
......
writing dependency_links to kfmd.egg-info/dependency_links.txt
writing top-level names to kfmd.egg-info/top_level.txt
writing manifest file 'kfmd.egg-info/SOURCES.txt'
error: package directory 'kfmd/openapi_client' does not exist
```

Since that's import by kfmd/metadata.py, personally I think we should check in the openapi_client code, at least we should add related reminding to new user to run `make swagger-py-client`, but if so seems it's not whole python pacakge.  so I think we should check in the openapi_client. Thanks.

**Comment by jinchihe** [comment]
@zhenghuiwang @gaoning777 What's you viewpoint? check in or update doc? personally I perfer to check in the `openapi_client` dir. Thanks.

**Comment by zhenghuiwang** [comment]
Thanks for raising this issues. I recommend use `pip install kfmd` to get the python library.

I need to update the documents and demo notebook to use pip install. /assign @zhenghuiwang 

I replied why it is not checked in https://github.com/kubeflow/metadata/pull/93#issuecomment-510961548

**Comment by jinchihe** [comment]
Thanks @zhenghuiwang, Yes, that's good for code maintenance. but now I'm thinking if also has problem for `pip install` without openapi_client since kfmd depends on that, you will generate that during installation?

**Comment by zhenghuiwang** [comment]
Thanks for filing this issue. The pip install problem is fixed for `kfmd` with version >= 0.1.7.

/close
